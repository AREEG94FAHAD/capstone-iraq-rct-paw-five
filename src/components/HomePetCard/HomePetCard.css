
.petLocation {
    position: absolute;
    background-color: rgba(255, 255, 255, 0.466);
    border-radius: 60px;
}

.detailsBtn {
    border-radius: 30px;
    width: 90px;
    background-color: #ffc533;
    color: white;
    transition: background 0.5s;
    transition:  box-shadow 0.5s;
    border: none;
}

.detailsBtn:hover,
.detailsBtn:active,
.detailsBtn:focus {
    background-color: #f85e49 !important;
    outline: 0 !important; 
}



