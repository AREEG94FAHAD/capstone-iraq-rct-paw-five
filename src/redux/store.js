import { createStore, applyMiddleware } from 'redux'
import { composeWithDevTools } from 'redux-devtools-extension'
import logger from 'redux-logger'
import combineReducers from './reducers/AllReducers'

const store = createStore(
  combineReducers,
  composeWithDevTools(applyMiddleware(logger))
)

export default store
