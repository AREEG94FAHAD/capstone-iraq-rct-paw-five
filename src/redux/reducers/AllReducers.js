import { combineReducers } from 'redux'
import petReducer, {
  langReducer,
  myPets,
  deleteMyPet,
  updateMyPet,
} from './petReducer'
import user from './user'

export default combineReducers({
  langReducer,
  pets: petReducer,
  user,
  myPets,
  deleteMyPet,
  updateMyPet,
})
