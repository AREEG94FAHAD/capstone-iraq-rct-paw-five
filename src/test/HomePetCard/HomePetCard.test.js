import React from 'react'
import renderer from 'react-test-renderer'
import { Provider } from 'react-redux'
import { createStore } from 'redux'
import { BrowserRouter as Router } from 'react-router-dom'
import HomePetCard from '../../components/HomePetCard/HomePetCard'
import AllReducers from '../../redux/reducers/AllReducers'
const store = createStore(AllReducers)

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <HomePetCard store={store} />
        </Router>
      </Provider>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
