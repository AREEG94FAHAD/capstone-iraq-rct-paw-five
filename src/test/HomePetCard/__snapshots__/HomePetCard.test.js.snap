// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  dir="ltr"
>
  <div
    className="w-100 mx-lg-5 m-md-3 m-sm-1 card my-3 position-relative card"
    style={
      Object {
        "maxHeight": "591px",
        "maxWidth": "360px",
      }
    }
  >
    <img
      className="card-img-top"
      src="undefined"
      style={
        Object {
          "height": "14rem",
        }
      }
    />
    <span
      className="position-absolute p-1"
      style={
        Object {
          "backgroundColor": "rgba(255, 255, 255, 0.466)",
          "borderRadius": "15px",
          "left": "220px",
          "top": "10px",
        }
      }
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-map-marker-alt fa-w-12 petLocationIcon"
        data-icon="map-marker-alt"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 384 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M172.268 501.67C26.97 291.031 0 269.413 0 192 0 85.961 85.961 0 192 0s192 85.961 192 192c0 77.413-26.97 99.031-172.268 309.67-9.535 13.774-29.93 13.773-39.464 0zM192 272c44.183 0 80-35.817 80-80s-35.817-80-80-80-80 35.817-80 80 35.817 80 80 80z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
       
       
    </span>
    <div
      className="card-body"
    >
      <div
        align="center"
        className="card-title h5"
        style={
          Object {
            "color": "#d18d28",
          }
        }
      />
      <p
        align="center"
        className="card-text"
      >
        
      </p>
      <span
        className="float-left"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-mars fa-w-12 petGenderIcon"
          data-icon="mars"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 384 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M372 64h-79c-10.7 0-16 12.9-8.5 20.5l16.9 16.9-80.7 80.7c-22.2-14-48.5-22.1-76.7-22.1C64.5 160 0 224.5 0 304s64.5 144 144 144 144-64.5 144-144c0-28.2-8.1-54.5-22.1-76.7l80.7-80.7 16.9 16.9c7.6 7.6 20.5 2.2 20.5-8.5V76c0-6.6-5.4-12-12-12zM144 384c-44.1 0-80-35.9-80-80s35.9-80 80-80 80 35.9 80 80-35.9 80-80 80z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
         
        <span
          className="p-1 text-danger"
        />
      </span>
      <a
        className="detailsBtn border-0 float-right btn btn-primary"
        disabled={false}
        href="/adopt/undefined"
        onClick={[Function]}
      >
        Details
      </a>
    </div>
  </div>
</div>
`;
